lib/togglv8/time_entries.rb

```diff
@@ -87,7 +87,25 @@ def get_time_entries(dates = {})
     def update_time_entries_tags(time_entry_ids, params)
       return if time_entry_ids.nil?
       requireParams(params, ['tags', 'tag_action'])
-      put "time_entries/#{time_entry_ids.join(',')}", { 'time_entry' => params }
+      time_entries = put "time_entries/#{time_entry_ids.join(',')}", { 'time_entry' => params }
+      return time_entries if params['tag_action'] == 'add'
+
+      #see https://github.com/toggl/toggl_api_docs/issues/200
+      time_entries = [time_entries] unless time_entries.is_a? Array
```

Comment:
I would prefer using a different variable name instead of overloading `time_entries`.